### PR TITLE
chore: reorganize `build` cli

### PR DESCRIPTION
**Pull request by ndonkoHenri**
## Summary by Sourcery

Enhancements:
- Refactor the build command in the Flet CLI to improve code organization and readability by splitting functionalities into smaller methods.

### CODE REVIEW

File: sdk/python/packages/flet-cli/src/flet_cli/commands/build.py [hunk 1]
```diff
@@ -49,28 +47,6 @@ class Command(BaseCommand):
...
-        self.python_module_name = None
-        self.get_pyproject = None
-        self.python_app_path = None
-        self.no_rich_output = None
-        self.emojis = {}
-        self.dart_exe = None
-        self.verbose = False
-        self.build_dir = None
-        self.flutter_dir: Optional[Path] = None
-        self.flutter_exe = None
```
Comment (ndonkoHenri) [inline sdk/python/packages/flet-cli/src/flet_cli/commands/build.py]: why remove this?

Comment (FeodorFitsner) [reply sdk/python/packages/flet-cli/src/flet_cli/commands/build.py]: It makes type checker really "red":

image

Comment (ndonkoHenri) [reply sdk/python/packages/flet-cli/src/flet_cli/commands/build.py]: The red lines you are showing seem to come from using `self.options.X` instead? 

Without that block of initializations i get: 
image

SO discussion: https://stackoverflow.com/a/19292653 (common python practice)

Comment (FeodorFitsner) [reply sdk/python/packages/flet-cli/src/flet_cli/commands/build.py]: Looks like it's coming from `self.options`. What would be the solution?

image

Comment (ndonkoHenri) [reply sdk/python/packages/flet-cli/src/flet_cli/commands/build.py]: Perhaps use `cast` as you did in the others? 
Or perhaps just mute the warning?

**Pull request merged**